rest_framework_jwt/utils.py

```diff
@@ -115,21 +127,30 @@ def jwt_decode_handler(token):
     )
 
 
-def jwt_response_payload_handler(token, user=None, request=None):
+def jwt_response_payload_handler(token, user=None, request=None, **kwargs):
     """
     Returns the response data for both the login and refresh views.
     Override to return a custom response such as including the
     serialized representation of the User.
 
     Example:
 
-    def jwt_response_payload_handler(token, user=None, request=None):
+    def jwt_response_payload_handler(token, user=None, request=None, **kwargs):
         return {
             'token': token,
             'user': UserSerializer(user, context={'request': request}).data
         }
 
     """
-    return {
+    data = {
```

Comment:
here you return token only, and earlier you return token along with user object. This is a reminder to remove user from above mentioned places.